integration-tests/feature_list.txt

```diff
@@ -37,8 +37,8 @@ features/three_scale.feature
 #features/api_backbone.feature
 features/gemini.feature
 features/resilient_infrastructure.feature
-features/three_scale_stack_analyses.feature
-features/three_scale_component_analyses.feature
+#features/three_scale_stack_analyses.feature
```

Comment:
three scale tests not required as we run everything on three scale now

Reply:
Can you also remove the code associated with this?